foundation/foundation-mda/src/main/resources/templates/data-delivery-pyspark/encryption.py.vm

```diff
@@ -78,13 +78,13 @@
 ## process a set[CustomRecord]
         # return type is a set
         return_payload = set([])
-        aiops_encrypt = AesCbcEncryptionStrategy()
+        aissemble_encrypt = AesCbcEncryptionStrategy()
         if(algorithm == 'VAULT_ENCRYPT'):
-            aiops_encrypt = VaultRemoteEncryptionStrategy()
+            aissemble_encrypt = VaultRemoteEncryptionStrategy()
 
         for record in inbound:
             for column in fields_to_update:
```

Comment:
S: this and the following `CustomRecord` change look off from a quick scan...

Reply:
I made these changes when my build was failing due to the Pylint implementation. I wasn't aware that I needed to run `git clean -dfx` in order to fix this issue. I reverted the change.